itenium-socks/src/app/sock-layout/sock-layout.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { Sock } from '../socks/sock.model';
+
+@Component({
+  selector: 'app-sock-layout',
+  standalone: true,
+  imports: [],
+  templateUrl: './sock-layout.component.html',
+  styleUrl: './sock-layout.component.scss'
+})
+export class SockLayoutComponent {
+    @Input() sockModel! : Sock;
```

Comment:
Is natuurlijk wat persoonlijke voorkeur en/of hangt af van project tot project maar ik zou niet expliciet `Model` toevoegen aan een input